docs/07-cli/hln/commands/hln.md

```diff
@@ -4,14 +4,14 @@ Heighliner: Cloud native best practices to build and deploy your applications
 
 ### Synopsis
 
-
+```
```

Comment:
This file is not editable. It is automatically generated by cobra and synced here: https://github.com/h8r-dev/heighliner-website/blob/main/.github/workflows/hln-commands.yml

Reply:
That's good suggestion. Created issue: https://github.com/h8r-dev/heighliner/issues/158

Let's close this PR and fix it in the syncer